lib/common/actions/index.js

```diff
@@ -131,14 +132,56 @@ function getErrorMessageFromJson (
     }
     // re-assign error message after it gets used in the detail.
     errorMessage = json.message || JSON.stringify(json)
+    if (json.detail && json.detail.includes('Referenced pattern IDs:')) {
+      // Custom formatting to link to pattern stops to remove if so desired.
+      // Pull the string of patternIds
+      const patternSplit = json.detail.split(/\[(.*?)\]/) // Match text within square brackets (our comma separated list of patternIds)
+      detail = patternSplit[0]
+      const patternsMatch = patternSplit[1]
+      if (patternsMatch) {
+        const patterns = patternsMatch.split(',')
+        // $FlowFixMe
+        detail = <PatternLinkErrorMessage detail={detail} patterns={patterns} />
+      }
+    }
     if (json.detail && json.detail.includes('conflicts with an existing trip id')) {
       action = 'DEFAULT'
     }
   }
-  console.error(errorMessage)
```

Comment:
Why did you remove this? I found it helpful to have the errors in the console

Reply:
Sorry, accident. Restored in https://github.com/ibi-group/datatools-ui/pull/943/commits/5a027d784a5ab84f546bc645f448d4771f77e9bc